README.md

```diff
@@ -212,9 +343,10 @@ and [development](https://docs.ansible.com/ansible/latest/dev_guide/index.html).
   modified with a module (e.g. the `ini` module) then default to using the `template` module. This both makes
   it easier to see what is being configured and to add additional, more complex options in the future.
 * Avoid the use of `lineinfile` wherever that might be feasible. Try editing files directly using other built
-  in modules (e.g. `ini_file`) or reading and parsing. If you are modifying more than a tiny number of lines
-  or in a manner more than trivially complex, try leveraging the `template` module, instead. This will allow
-  the entire structure of the file to be seen by later users and maintainers.
+  in modules (e.g. `ini_file`, `blockinfile`, `xml`) or reading and parsing. If you are modifying more than a
+  tiny number of lines or in a manner more than trivially complex, try leveraging the `template` module,
+  instead. This will allow  the entire structure of the file to be seen by later users and maintainers. The
+  use of `lineinfile` should include a comment with justification.
```

Comment:
Since you are already changing the `lineinfile` section, please consider including https://github.com/oasis-roles/meta_standards/pull/22 in this PR.